zstacklib/zstacklib/utils/iproute.py

```
@@ -692,10 +758,11 @@ def delete_namespace(namespace):
     try:
         pyroute2.netns.remove(namespace)
     except OSError as e:
-        if e.errno == os.errno.ENOENT: # no namespace found
+        if e.errno == os.errno.ENOENT:  # no namespace found
             raise NoSuchNamespace(namespace)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)